apps/frontend/src/pages/quiz/libs/components/quiz-form/quiz-form.tsx

```diff
@@ -2,17 +2,17 @@ import { QuizQuestion } from "~/libs/components/components.js";
 import { useAppForm } from "~/libs/hooks/hooks.js";
 
 type FormValues = {
-	value: string;
+	answer: "";
```

Comment:
```suggestion
[key: string]: number;
```
Shouldn't it be like this? The value for each answer is id, isn't it?